code/graphics/opengl/gropenglshader.cpp

```diff
@@ -31,6 +31,18 @@
 
 SCP_vector<opengl_shader_t> GL_shader;
 
+typedef std::pair<int, uint32_t> shader_descriptor_t;
```

Comment:
Should this be int32_t too?

Reply:
If you want you can use fixed size types wherever you want but it's not required unless the type name is very long (such as `unsigned int` or `unsigned long long`).

Reply:
won't then, I better understand the coding convention now.